backend/auth/auth.js

```diff
@@ -130,40 +71,135 @@ exports.userById = async (req, res, next) => {
 };
 
 /**
- * Hairdresser id rating route
+ * Update user
  */
-exports.ratings = async (req, res, next) => {
+exports.updateUser = async (req, res, next) => {
+	const token = req.headers.authorization.split(' ')[1];
+	const payload = jwt.tokenPayload(token);
+	const time = new Date().getTime();
+
+	if (time < payload.exp) {
+		const updateUserId = req.params.id;
+		const id = req.user._id.toString();
```

Comment:
`_id ` is unknown to frontend. User model does not contain `_id`, hence user can't set rating. Currently email (available in JWT) is unique identifier used in frontend.

Frontend and backend need to be unambiguous on how to identify users.